### ISSUE TITLE
reset chapter state

### ISSUE THREAD

**Issue by Guy27** [TortugaPower/BookPlayer#349]
please allow to reset "heard" status for all chapters of a book at once (playlist).

**Comment by pichfl** [comment]
Thanks for bringing this up. This is an oversight from when we separate playback position from finished state and will be added.

**Comment by GianniCarlo** [comment]
This can already be done though with the latest version in the appstore. You have to swipe left on the playlist item in the table, and from the options you can affect all the items if you press 'Mark as unfinished'

**Comment by GianniCarlo** [comment]
What we should do and have had talks about it, is evolve the bulk edit to provide more options instead of just sort | move | trash. But for the time being this can already be accomplished for individual playlists

**Comment by pichfl** [comment]
@GianniCarlo as I wrote: Mark as unfinished does not reset playback positions only the finished state.

**Comment by GianniCarlo** [comment]
Oh right, I misunderstood the intent of the issue 😅. Mmmmm the separation thing was intended like it is now. If you want to reset playback, there's an option 'Jump to start' which if applied  in the way I described before, it'll reset all the playlist's items playback position

**Comment by pichfl** [comment]
I'm the one who missed your point, you're of course right. Maybe we need to rename "jump to start" for playlists to "Reset playback position"?

**Comment by GianniCarlo** [comment]
yeah that could be a good option, 'jump to start' certainly isn't conveying well its message

**Comment by Guy27** [comment]
thanks guys, i agree with the proposed solution. it will be much clearer
